projects/plugins/jetpack/modules/search/instant-search/lib/api.js

```diff
@@ -210,14 +210,7 @@ function generateApiQueryString( {
 
 	switch ( resultFormat ) {
 		case 'product':
-			fields = fields.concat( [
-				'meta._wc_average_rating.double',
-				'meta._wc_review_count.long',
-				'wc.currency_position',
-				'wc.currency_symbol',
-				'wc.price',
-				'wc.sale_price',
-			] );
+			fields = fields.concat( [ 'meta._wc_average_rating.double', 'meta._wc_review_count.long' ] );
```

Comment:
Do we need to keep those maybe, since they're needed for the Product views?

Reply:
We'd originally removed these fields in this PR since the API specification was still in flux for WooCommerce pricing data. Due to this situation, the API would throw 400 errors and prevent the product result layout from showing any results.


Now that #18225 has shipped, this change has been dropped entirely from this PR 🎆